tensorflow/contrib/learn/python/learn/estimators/dnn.py

```diff
@@ -124,17 +129,23 @@ def _dnn_model_fn(features, labels, mode, params):
   hidden_layer_partitioner = (
       partitioned_variables.min_max_variable_partitioner(
           max_partitions=num_ps_replicas))
+
   for layer_id, num_hidden_units in enumerate(hidden_units):
     with variable_scope.variable_scope(
         parent_scope + "/hiddenlayer_%d" % layer_id,
         values=[net],
         partitioner=hidden_layer_partitioner) as scope:
+
       net = layers.fully_connected(
           net,
           num_hidden_units,
           activation_fn=activation_fn,
-          variables_collections=[parent_scope],
+#HL think we can add here the batch_norm.
```

Comment:
I don't think this comment is needed (and in general one-line comments should have the same indentation as other code). 
Also please format the arguments the same way as others (e.g. no extra spaces).